app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,26 @@
+class SessionsController < ApplicationController
+
+  before_action :find_email_downcase, only: [:create]
```

Comment:
Rubocop báo đoạn đấy bị lỗi "Assignment Branch Condition size for create is too high"  nên em phải tách ra, không viết trực tiếp trong hàm được.